scripts/camera/follow-camera.js

```diff
@@ -15,7 +15,7 @@ FollowCamera.attributes.add('lerpAmount', {
     type: 'number',
     min: 0,
     max: 1,
-    default: 0.1,
+    default: 0.99,
```

Comment:
Are you sure this should be 0.99? The attribute description says:

> The amount to lerp the camera towards its desired position every frame

But if the camera lerps 0.99 towards the target after the first frame, you're pretty much already at your destination.

Reply:
It doesn't lerp 99% on the first frame as it's multiplied by a power of dt:

```
this.currentPos.lerp(this.currentPos, this.targetPos, 1 - Math.pow(1 - this.lerpAmount, dt));
```

Reply:
Cool, but then isn't the attribute description text misleading?

Reply:
That description should change though, that said 🤔

Reply:
Added a description change suggestion :)

Reply:
I'm gonna let @slimbuck approve this one - my maths sucks.